src/pages/team.js

```diff
@@ -1,12 +1,208 @@
+    title: "Vasu Zalawadia (Co-Director of Research)",
+    image: Vasu,
+    link: "https://www.linkedin.com/in/vzalawadia/",
+    id: 7,
+  },
+  {
+    description: "[Insert]",
+    title: "Nishay Madhani (Director of Technology)",
+    image: Nishay,
+    link: "https://nshmadhani.com",
+    id: 8,
+  },
+  {
+    description:
+      "Ivan is a 2nd year Computer Science & Mathematics major. He is into Soccer and DeFi. His favorite crypto is Ethereum.",
+    title: "Ivan Rosales (Director of Governance)",
+    image: Ivan,
+    link: "https://www.linkedin.com/in/ivangrosales/",
+    id: 9,
+  },
+  {
+    description: "[Insert]",
+    title: "Bernice Luong (Director of Marketing)",
+    image: Bernice,
+    link: "https://www.linkedin.com/in/bernice-luong/",
+    id: 10,
+  },
+  {
+    description: "[Insert]",
+    title: "Megan Gilbert (Head of Social Media)",
+    image: Megan,
+    link: "https://www.linkedin.com/in/megan--gilbert/",
+    id: 11,
+  },
+  {
+    title: "Shubh Patni (Head of Public Relations)",
+    image: Shubh,
+    description:
+      "Shubh Patni is a first year student studying CS and Economics. He is a Blogger, Programmer, and creates content related to programming and tech. He is working at Phemex Student Fellowship and Coinbase Analyst Program. His Favorite Crypto is Ethereum",
+    link: "https://shubhpatni.com",
+    id: 12,
+  },
+  {
+    description: "[Insert]",
+    title: "Luke O'Neill (Director of Operations)",
+    image: Luke,
+    link: "https://www.linkedin.com/in/lukefoneill",
+    id: 13,
+  },
+];
+
+
+const renderCard = (card, index) => {
+
+    return(
+        
+        <Card border="danger" style={{ width: '20rem' }} text="dark" key={index} class="box">
+      <Card.Img variant="top" src= {card.image} class="cardimg" />
+      <Card.Body>
+        <Card.Title>{card.title}</Card.Title>
+        <Card.Text>
+          {card.description}
+        </Card.Text>
+        <Button href={card.link} variant="danger">Social</Button>
+      </Card.Body>
+    </Card>
+
+    )
+}
+
+/*
+function checkID(thisId, targetId) {
+
+    return thisId < targetId;
+}
+*/
 
 function Team() {
     return (
+        <section>
         <AppLayout>
             <h2>Team</h2>
+            <div class="circle-wrapper">
+    <div class="success rectangle"></div>
+  </div>
+            <Container>
+                <Row>
+                
+                <Col >
+                {cardInfo.slice(0,2).map(renderCard)} 
+                {cardInfo.slice(2,4).map(renderCard)}
+                </Col>
+               
+               <Col>
+                {cardInfo.slice(4,6).map(renderCard)}
+                {cardInfo.slice(6,8).map(renderCard)}
+                </Col>
+                
+
+                <Col>
+                {cardInfo.slice(8,10).map(renderCard)}
+                {cardInfo.slice(10,12).map(renderCard)}
+                
+                {cardInfo.slice(12,13).map(renderCard)}
```

Comment:
This works, however it isn't extensible for when new members are added. We only want to have to change/add to member.json and not have to add new cards manually here. What you should do is iterate through the cardInfo array and grab two (or four, or however many) and add a new <Col/> to a new array, which is then rendered dynamically.

Reply:
So, above the return statement, you'll need to create a function to create this array of Cols and then call that function within the return Row

Something like this:

`
    const renderCardGrid = () => {
      cardGrid = []

     let i = 0;
     while (i < cardInfo.length) {
         // grab i and i+1 from cardInfo array and render 2 at a time (youll have to check for the end of the array in the case of odd-lengthed cardInfo list
         cardGrid.push(<Col> 
                renderCard(cardInfo[i])
                renderCard(cardInfo[i+1])
         </Col>) 
         i += 2;
    }
 }
`

Then within the <Row> block starting on line 162, you will replace everything with `<Row>{renderCardGrid()}</Row>`